packages/genesys-spark-components/src/components/beta/gux-chat-input/i18n/en.json

```diff
@@ -0,0 +1,4 @@
+{
+  "inputPlaceholder": "Ask Genesys AI",
+  "cta": "Generate Genesys AI text"
```

Comment:
This is minor and not sure if we do have naming conventions for keys but should do you think something like `generateGenesysAIText` may be better as if you read where you are calling it in the code its just `this.i18n('cta')` which may to be clear.